scripts/rebuild-testbench.sh

```diff
@@ -5,10 +5,16 @@
 # fail on any errors
 set -e
 
+# Defaults
+BUILD_TYPE=native
+BUILD_DIR_NAME=build_testbench
+BUILD_TARGET=install
```

Comment:
I thought `install` was not needed?

Reply:
I kept "install" for native build, then the executable and the libraries for x86-64 version remain unchanged. The BUILD_TARGET is cleared for xtensa build.